demo/web/src/main/notebooks/A. Real-time table ops/A3 Do time series and relational joins.md

```diff
@@ -0,0 +1,49 @@
+# Time series and relational joins
+
+In our previous notebook, we showed how to filter and decorate our time tables. In this notebook, we show how to perform joins with our time series data.
+
+Let's start again by simulating a year's worth of daily measurements, but this time we will have two tables with slightly different timestamps. This is great for a simulation because there's no guarantee that a real example will collect data with exact timestamp matches.
```

Comment:
```suggestion
Let's start again by simulating a year's worth of daily measurements, but this time using two tables with slightly different timestamps. This is great for a simulation because there's no guarantee that a real example will collect data with exact timestamp matches.
```